src/avfoundation.cs

```diff
@@ -1023,6 +1023,32 @@ interface AVAudioEngine {
 		[Watch (4, 0), TV (11, 0), Mac (10, 13), iOS (11, 0)]
 		[Export ("manualRenderingSampleTime")]
 		long ManualRenderingSampleTime { get; }
+
+#if !WATCH
+		[TV (12,0), Mac (10,14, onlyOn64: true), iOS (12,0), NoWatch]
+		[Export ("connectMIDI:to:format:block:")]
+		void ConnectMIDI (AVAudioNode sourceNode, AVAudioNode destinationNode, [NullAllowed] AVAudioFormat format, [NullAllowed] AUMidiOutputEventBlock tapBlock);
```

Comment:
did we use `MIDI` or `Midi` in existing API ?
The later is what the FXDG suggest but we should be consistent

Reply:
@dalexsoto we already have params with that, name, Shouldn't we be consistent? I have no personal preference.

Reply:
My take would be to do the right thing now for new stuff since it helps to be self documented and we will need to fix less of this in the future.